test/focus-group-polyfill.test.ts

```diff
@@ -0,0 +1,477 @@
+import { JSDOM } from 'jsdom'
```

Comment:
Can we add a few tests for `focusGroupPolyfill` and `focusGroupKeyUX` to be sure that there are no conflicts between them?

Reply:
Yes, sure!